qutebrowser/completion/models/miscmodels.py

```diff
@@ -103,17 +103,21 @@ def session(*, info=None):
     return model
 
 
-def _tabs(*, win_id_filter=lambda _win_id: True, add_win_id=True):
+def _tabs(*, win_id_filter=lambda _win_id: True, add_win_id=True, cur_win_id=0):
     """Helper to get the completion model for tabs/other_tabs.
 
     Args:
         win_id_filter: A filter function for window IDs to include.
                        Should return True for all included windows.
         add_win_id: Whether to add the window ID to the completion items.
+        cur_win_id: Window ID to be passed from info.win_id
     """
     def delete_tab(data):
         """Close the selected tab."""
-        win_id, tab_index = data[0].split('/')
+        win_id = cur_win_id
+        # data[0] can be 'tabInd' or 'winID/tabInd'
+        tab_index = data[0].split('/')[-1]
```

Comment:
I think we still need to get a window ID from `data`, and only fall back to `cur_win_id` if the former is missing. Otherwise we might delete the wrong tab if there are multiple windows.